### DISCUSSION TITLE
Tweet the latest release of the syrius Mobile Wallet

### DISCUSSION THREAD

**Discussion by 0x3639** [coinselor/twitter#5] [Tweet Ideas]
🚀 Exciting News! The updated syrius Mobile Wallet for @Zenon_Network is ready for beta testing.  Download the latest release here: https://forum.zenon.org/t/syrius-wallet-mobile-app/659/125?u=0x3639

**Comment by sugoibtc** [comment]
Maybe we could give it some more context? I tried this format and it should fit within the character limit on X.

🚀 Exciting news! The Syrius Mobile Wallet v0.0.2 is now live for testing 👽

Functionalities:
- Create & import wallet
- Send & receive transactions
- Generate plasma using PoW or by fusing QSR
- Wallet settings
- Staking
- Delegating

Download here 👇
https://forum.zenon.org/t/syrius-wallet-mobile-app/659/125?u=0x3639

**Reply by 0x3639** [reply]
much better!!  maybe the bullets are green checks.  not sure we can do that.  I can try in my test account.

**Reply by sugoibtc** [reply]
That would be possible, just checked it! This is the final draft, containing the correct version of the wallet and replaced the functionality "Wallet settings" with WalletConnect. 

Updated tweet:

Exciting news! The Syrius Mobile Wallet v0.1.0 is now LIVE for testing 👽

Functionalities:
✅Create & import wallet
✅Send & receive transactions
✅Generate plasma using PoW/fusing QSR
✅WalletConnect
✅Delegating & staking
✅Notifications

Download here: https://forum.zenon.org/t/syrius-wallet-mobile-app/659/125?u=0x3639
